src/backend/src/prisma/manual.ts

```diff
@@ -142,19 +142,30 @@ const migrateToProposedSolutions = async () => {
 };
 
 /**
- * Manually create a team via direct db.
+ * Manually create two team via direct db. (for dev use only lol)
```

Comment:
lol this should be deleted and replaced with seed data

Reply:
https://github.com/Northeastern-Electric-Racing/FinishLine/issues/164

Reply:
true